app/javascript/stylesheets/active_admin.scss

```diff
@@ -8,8 +8,8 @@
 // $sidebar-width: 242px;
 
 // Active Admin's got SASS!
-@import "active_admin/mixins";
-@import "active_admin/base";
+@import "~@activeadmin/activeadmin/src/scss/mixins";
+@import "~@activeadmin/activeadmin/src/scss/base";
```

Comment:
I know this was autogenerated by active admin and I'm not sure if it's possible what I'm going to ask you but, could you try removing CSS/SCSS from `app/javascript/*` and try to use the asset pipeline for CSS (you might need to restore some active admin CSS files in `app/assets/stylesheets/*`?

Reply:
Yeah, sorry I misunderstood you!